azkaban-common/build.gradle

```diff
@@ -1,10 +1,5 @@
 configurations {
-  all {
-    transitive = false
-  }
-  errorprone {
-    transitive = true
-  }
+  errorprone
```

Comment:
Actually, this is no longer required. removing it.

Since the errorprone plugin is applied to all subprojects, we do not need any custom configurations for plugin integration. This was required for previous versions of the plugin / gradle.